Examples/MagicWeather/MagicWeather/Constants.swift

```diff
@@ -16,13 +16,23 @@ struct Constants {
     /*
      The API key for your app from the RevenueCat dashboard: https://app.revenuecat.com
      */
-    #error("Modify this property to reflect your app's API key, then comment this line out.")
-    static let apiKey = "REVENUECAT_API_KEY"
-    
+    static let apiKey: String = {
+        // return "your-api-key"
+        guard let key = Bundle.main.object(forInfoDictionaryKey: "EXAMPLE_APP_API_KEY") as? String, !key.isEmpty else {
```

Comment:
I believe there was some discussion about this already, but I wonder if it would be better for the sample app to have the key hardcoded in code instead of obtained from the bundle, to follow best practices... But as discussed, maybe it's ok since this is after all a "public" api key.

Reply:
Fair point, yes.
This is why I left the commented out code
```swift
// return "your-api-key"
```
And also why I separated this from #4795, because it's the same idea but with the caveat of being the example project for integrators of the SDK.
I really don't have a strong opinion about this one, but this is worth discussing, yes.